httpstan/compile.py

```diff
@@ -0,0 +1,37 @@
+"""Runs the `stanc` binary in a subprocess to compile a Stan program."""
+import os
+import subprocess
+import tempfile
+
+
+def compile(program_code: str, stan_model_name: str) -> str:
+    """Return C++ code for Stan model specified by `program_code`.
+
+    Arguments:
+        program_code
+        stan_model_name
+
+    Returns:
+        str: C++ code
+
+    Raises:
+        ValueError: Syntax or semantic error in program code.
+
+    """
+    with tempfile.NamedTemporaryFile() as fh:
+        fh.write(program_code.encode())
```

Comment:
Is the default encoding utf-8?

Can Stan handle utf-16? (Japanese users for example might want to have this feature)

(In comments)

Reply:
I know there was a question about this at some point. In any event, this is an issue for the Stanc3 developers. The interfaces will not have any control over what encodings are acceptable by the compiler.